package.json

```
@@ -123,7 +123,12 @@
     "react-native-ble-plx": "3.1.2",
     "**/babel-runtime/regenerator-runtime": "^0.13.8",
     "redux-persist-filesystem-storage/react-native-blob-util": "^0.19.9",
-    "xmldom": "npm:@xmldom/xmldom@0.7.13"
+    "xmldom": "npm:@xmldom/xmldom@0.7.13",
+    "**/@walletconnect/socket-transport/ws": "7.5.10",
+    "**/@ethersproject/providers/ws": "7.5.10",
+    "**/puppeteer-core/ws": "8.17.1",
+    "react-native/ws": "6.2.3",
+    "socket.io-client/engine.io-client/ws": "^8.17.1"
```

```suggestion
    "socket.io-client/engine.io-client/ws": "^8.17.1",
    "**/appium-base-driver/webdriverio/puppeteer-core/**/ws": "7.5.10",
```
This will resolve ^7.2.3 of ws to 7.5.10